src/unit_tests/shared/test_syscheck_op.c

```diff
@@ -262,21 +312,30 @@ static void test_remove_empty_folders_success(void **state) {
 #ifndef TEST_WINAGENT
     char *input = "/var/ossec/queue/diff/local/test-dir/";
     char *first_subdir = "/var/ossec/queue/diff/local/test-dir";
+    char *second_subdir = "/var/ossec/queue/diff/local";
 #else
     char *input = "queue/diff\\local\\test-dir\\";
     char *first_subdir = "queue/diff\\local\\test-dir";
+    char *second_subdir = "queue/diff\\local";
 #endif
     int ret = -1;
     char message[OS_SIZE_1024];
+    char **not_null;
```

Comment:
Please rename this variable to something a little more descriptive like `mock_directory_content`